src/graphql/Quote.graphql

```diff
@@ -30,6 +30,20 @@ query Quote($id: ID!) {
           currency
         }
       }
+      perils(locale: sv_SE) {
```

Comment:
Some idea on how we should we set/get locale in the Quote query? @palmenhq

Reply:
I think the best option we have is to pick it from the url. I think we could use it as an input (`$perilsLocale`) to the query, and then pick the locale from [`useCurrentLocale`](https://github.com/HedvigInsurance/web-onboarding/blob/master/src/components/utils/CurrentLocale.tsx). In there you even have a `getLocaleIsoCode` which i _think_ you can use right off the bat

Reply:
Let's do it!